lib/unit-ruby/deposit_account.rb

```diff
@@ -24,5 +24,51 @@ class DepositAccount < APIResource
     include ResourceOperations::Create
     include ResourceOperations::Save
     include ResourceOperations::Find
+
+    def resource_path
+      self.class.resource_path(id)
+    end
+
+    def close (reason)
+      updated_resource = self.class.connection.post(
+        "#{resource_path}/close", 
+        {
+          data: {
+            type: "depositAccountClose",
+            attributes: {
+              reason: reason,
+            }.compact
+          }
+        }
+      )
+
+      update_resource_from_json_api(updated_resource)
+    end
+
+    def freeze (reason, reasonText)
```

Comment:
Same spacing nit + using snake_case instead of camelCase:
```suggestion
    def freeze(reason, reason_text)
```